### ISSUE TITLE
Chatbidding

### ISSUE THREAD

**Issue by marjorg** [Mordonus/RaidOps#459]
(Originally posted as comment on Curse)
Whenever we try to distribute loot based on what was selected it crashes. Which means that we have to manually calculate the EP and distribute the loot if it's for off or slight.

**Comment by Drutol** [comment]
Hmm I found a few small issues concerning chatbidding , but assigning didn't lead to major crash , can you post the error message that W\* is displaying?

**Comment by marjorg** [comment]
We are raiding tonight, I'll update with error msg if we get it.

**Comment by marjorg** [comment]
This is the one from assigning loot:
...pData\Roaming\NCSOFT\WildStar\Addons\RaidOps\BID.lua:1011: attempt to index global 'selectedItem' (a nil value)
stack trace:
    ...pData\Roaming\NCSOFT\WildStar\Addons\RaidOps\BID.lua:1011: in function <...pData\Roaming\NCSOFT\WildStar\Addons\RaidOps\BID.lua:994>

We also got this when the loot window appeared:
...pData\Roaming\NCSOFT\WildStar\Addons\RaidOps\BID.lua:2342: attempt to index field 'Bid2' (a nil value)
stack trace:
    ...pData\Roaming\NCSOFT\WildStar\Addons\RaidOps\BID.lua:2342: in function 'Bid2CloseOnAssign'
    ...pData\Roaming\NCSOFT\WildStar\Addons\RaidOps\DKP.lua:1685: in function <...pData\Roaming\NCSOFT\WildStar\Addons\RaidOps\DKP.lua:1632>
    [C]: in function 'PostOnChannel'
    ui\ChatLog\ChatLog.lua:2046: in function <ui\ChatLog\ChatLog.lua:2045>
    [C]: in function 'Event_FireGenericEvent'
    ui\MasterLoot\MasterLoot.lua:399: in function <ui\MasterLoot\MasterLoot.lua:398>

**Comment by Drutol** [comment]
Oooh , I got it

**Comment by marjorg** [comment]
Is it a complicated thing to fix?

**Comment by Drutol** [comment]
Nope , something like big typo. I haven't checked it in-game yet but it's bound to be it ^^.

**Comment by Drutol** [comment]
line 1011 in BID.lua should look like this

```
        self.tPopUpItemGPvalues[self.wndBid:GetData().itemDrop:GetName()] = price
```

**Comment by marjorg** [comment]
Tried to apply the fix and reloadui. But in the addonlist RaidOps now has a red light in front of it and I get this error:

..AppData\Roaming\NCSOFT\WildStar\Addons\RaidOps\BID.lua:1: '=' expected near ''

**Comment by marjorg** [comment]
On another note. I just tried Network bidding and got the following error after selecting an item and clicking "Network bidding":

...pData\Roaming\NCSOFT\WildStar\Addons\RaidOps\BID.lua:1883: bad argument #1 to 'sub' (string expected, got nil)
stack trace:
    [C]: in function 'sub'
    ...pData\Roaming\NCSOFT\WildStar\Addons\RaidOps\BID.lua:1883: in function 'BidAddNewAuction'
    ...pData\Roaming\NCSOFT\WildStar\Addons\RaidOps\BID.lua:590: in function <...pData\Roaming\NCSOFT\WildStar\Addons\RaidOps\BID.lua:586>

**Comment by Drutol** [comment]
Which ML addon do you use? MasterLootDependency?

**Comment by marjorg** [comment]
Yes

**Comment by Drutol** [comment]
Try this BID.lua : http://pastebin.com/uJqCj9mz

**Comment by Drutol** [comment]
With Networkbidding fix : http://pastebin.com/zSFBD6uy

**Comment by marjorg** [comment]
Chatbidding seems to work fine now! But for networkbid I got this while trying to start a bidding round (after clicking "network bidding")

...pData\Roaming\NCSOFT\WildStar\Addons\RaidOps\BID.lua:1708: bad argument #1 to 'sub' (string expected, got nil)
stack trace:
    [C]: in function 'sub'
    ...pData\Roaming\NCSOFT\WildStar\Addons\RaidOps\BID.lua:1708: in function 'Bid2SendAuctionStartMessage'
    ...pData\Roaming\NCSOFT\WildStar\Addons\RaidOps\BID.lua:1939: in function <...pData\Roaming\NCSOFT\WildStar\Addons\RaidOps\BID.lua:1934>

**Comment by Drutol** [comment]
Fixed 3.08k.
